third_party/uvwasi/CMakeLists.txt

```diff
@@ -42,6 +42,12 @@ if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
   list(APPEND uvwasi_defines _GNU_SOURCE _POSIX_C_SOURCE=200112)
 endif()
 
+if (WALRUS_MODE STREQUAL "debug")
+    list(APPEND uvwasi_defines UVWASI_DEBUG_LOG)
+    add_compile_options( -O0 )
```

Comment:
Do you need two steps?

Reply:
No, it did not occur to me that it can take multiple options at the same time.